docs/source/error-codes/self-service/index.rst

```diff
@@ -11,81 +11,185 @@ Self-Service Error Codes (Experimental)
-+-----------------------------------------------+--------------------------------+--------------------------------------+-------------------------------------------------------------------------------------------------------------------------+---------------------------------------------+
-|TransactionService.getTransactionTrees         |NOT_FOUND                       |OUT_OF_RANGE                          |OUT_OF_RANGE can now be returned when attempting to access the data that has already been pruned.                        |PARTICIPANT_PRUNED_DATA_ACCESSED             |
-+-----------------------------------------------+--------------------------------+--------------------------------------+-------------------------------------------------------------------------------------------------------------------------+---------------------------------------------+
-|TransactionService.getTransactions             |INVALID_ARGUMENT                |OUT_OF_RANGE                          |OUT_OF_RANGE can now be returned when the supplied offset was out of range.                                              |REQUESTED_OFFSET_OUT_OF_RANGE                |
-+-----------------------------------------------+--------------------------------+--------------------------------------+-------------------------------------------------------------------------------------------------------------------------+---------------------------------------------+
-|TransactionService.getTransactions             |NOT_FOUND                       |OUT_OF_RANGE                          |OUT_OF_RANGE can now be returned when attempting to access the data that has already been pruned.                        |PARTICIPANT_PRUNED_DATA_ACCESSED             |
-+-----------------------------------------------+--------------------------------+--------------------------------------+-------------------------------------------------------------------------------------------------------------------------+---------------------------------------------+
+The table below outlines generic gRPC error code changes pertaining to the Ledger API
+and apply to all ledger backends. For changes specific to a ledger backend, check the next subsections.
+
++--------------------------------------------------+---------------------------------+---------------------------------------+-------------------------------------------------------------------------------------------------------------------------+---------------------------------------------+
+|Service method                                    |gRPC status code (legacy errors) |gRPC status code (self-service errors) |Remarks                                                                                                                  |Sef-service error code id                    |
```

Comment:
It would make sense to make the before/after state more explicit.
* 'Legacy' is hard to understand without context, hence, adding the explicit SDK version may help (such as `gRPC status code (before v1.18)` or similar).
* If self-service error codes need to be explicitly enabled, we should state that (e.g., `gRPC status code (self-service error codes enabled)`). In case self-service error codes are enabled by default, a shorter column name could be `gRPC status code (current)`.

Reply:
Makes sense. Will incorporate